src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToHtml5LocalDateTimeTransformer.php

```diff
@@ -22,6 +22,23 @@ class DateTimeToHtml5LocalDateTimeTransformer extends BaseDateTimeTransformer
 {
     const HTML5_FORMAT = 'Y-m-d\\TH:i:s';
 
+    private $strict;
+
+    public function __construct(string $inputTimezone = null, string $outputTimezone = null)
+    {
+        parent::__construct($inputTimezone, $outputTimezone);
+
+        if (2 < \func_num_args()) {
+            $this->strict = \func_get_arg(2);
```

Comment:
no need to to use `func_get_args`. Adding an optional argument in a constructor is not a BC break (a child constructor does not need to have a compatible signature)